x-pack/plugins/apm/common/processor_event.ts

```diff
@@ -9,8 +9,6 @@ export enum ProcessorEvent {
   error = 'error',
   metric = 'metric',
   span = 'span',
-  onboarding = 'onboarding',
-  sourcemap = 'sourcemap',
```

Comment:
We never query these so might as well remove them. For datastreams APM Server will no longer send onboarding or sourcemap documents.